workflow-core/src/commonMain/kotlin/com/squareup/workflow1/SessionWorkflow.kt

```diff
@@ -70,7 +70,29 @@ public abstract class SessionWorkflow<
    *     or [Flow.onCompletion][kotlinx.coroutines.flow.onCompletion] to handle that.
    *
    *     If you have a general cleanup operation that is fast and thread-safe then you could use
-   *     [Job.invokeOnCompletion][kotlinx.coroutines.Job.invokeOnCompletion].
+   *     [Job.invokeOnCompletion][kotlinx.coroutines.Job.invokeOnCompletion]. Otherwise use
+   *     [awaitCancellation][kotlinx.coroutines.awaitCancellation] with a finally block. Examples:
+   *
+   *     override fun initialState(
```

Comment:
You need to indent this more to get it to format as a code block. Use Ctrl-J to preview.

Reply:
Note that the existing codeblocks are a mess

Reply:
Ok, fixed both.